pwiz_tools/Skyline/Alerts/MultiButtonMsgDlg.cs

```diff
@@ -95,6 +95,46 @@ public MultiButtonMsgDlg(Control ctl, string btnText) : this(string.Empty, btnTe
             splitContainer.Panel1.Controls.Add(ctl);
         }
 
+        /// <summary>
+        /// Show a message box with a set of standard buttons, and an optional custom title.
+        /// </summary>
+        public static DialogResult Show(IWin32Window parent, string message, MessageBoxButtons buttons, string title = null)

```

Comment:
I would like to get rid of the "title" string. I didn't see any necessary uses of it. In fact, forcing us to look at it probably turned up 2 bugs.